sycl/include/CL/sycl/detail/pi.h

```diff
@@ -34,9 +34,10 @@
 //   pi_device_binary_property_set PropertySetsBegin;
 //   pi_device_binary_property_set PropertySetsEnd;
 // 2. A number of types needed to define pi_device_binary_property_set added.
+// 3. Added new ownership argument to piextContextCreateWithNativeHandle.
 //
-#define _PI_H_VERSION_MAJOR 2
-#define _PI_H_VERSION_MINOR 3
+#define _PI_H_VERSION_MAJOR 3
+#define _PI_H_VERSION_MINOR 4
```

Comment:
Could you please clarify why minor is not "0" after major version is increased?

Reply:
This is following the rules set in lines 25-28:

``` C++
// Every single change in PI API should be accompanied with the minor
// version increase (+1). In the cases where backward compatibility is not
// maintained there should be a (+1) change to the major version in
// addition to the increase of the minor.
```

Reply:
Ok, thanks.